core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -604,5 +623,44 @@ public static void validate() {
     checkTieredLocality();
   }
 
+  /**
+   * @return a list of worker-related configurations
+   */
+  public static List<ConfigProperty> getWorkerConfiguration() {
+    List<ConfigProperty> configInfoList = new ArrayList<>();
+    for (Map.Entry<String, String> entry : toMap().entrySet()) {
+      String keyName = entry.getKey();
+      if (isWorkerRelated(keyName)) {
+        Source source = getSource(PropertyKey.fromString(keyName));
+        String sourceStr;
+        if (source == Source.SITE_PROPERTY) {
+          sourceStr =
+              String.format("%s (%s)", source.name(), getSitePropertiesFile());
```

Comment:
we have similar logic in other places for converting a source to a sourceStr. Can we consolidate them?

Reply:
Create method getFormattedSource(PropertyKey key) to consolidate duplicate codes. thanks!